rdt/transformers/pii/anonymizer.py

```diff
@@ -244,6 +248,7 @@ def _fit(self, columns_data):
             data (pandas.Series):
                 Data to fit the transformer to.
         """
+        self.faker.seed_instance(self._faker_random_seed)
```

Comment:
I think that we should compute `self._faker_random_seed` here as well, since this is not calling `super()`